src/main/java/io/aiven/kafka/auth/audit/UserOperation.java

```diff
@@ -46,9 +46,7 @@ public boolean equals(final Object o) {
             return false;
         }
         final UserOperation that = (UserOperation) o;
-        return hasAccess == that.hasAccess
-            && Objects.equals(operation, that.operation)
-            && Objects.equals(resource, that.resource);
+        return hasAccess == that.hasAccess;
```

Comment:
This is wrong. It should use 3 fields for equals

Reply:
Yes. Fixing.